### ISSUE TITLE
Weeds Classification

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#75]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Weeds Classification</br>
:red_circle: **Aim** : An image processing model to identify the different breeds of weeds.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/imsparsh/deepweeds</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by taruarora** [comment]
I would like to work on this issue

**Comment by abhisheks008** [comment]
✅ To be Mentioned while taking the issue :

Full name :
GitHub Profile Link :
Email ID :
Participant ID (if applicable):
Approach for this Project :
What is your participant role? (Mention the Open Source program)

Please provide the necessary details
@taruarora

**Comment by taruarora** [comment]
Issue#75
Weeds classification
Name: Taru Arora
Email id: taruarora2605@gmail.com
GitHub : https://GitHub.com/taruarora
EMail id : taruarora2605@gmail.com
Approach :using CNN

**Comment by abhisheks008** [comment]
Try to implement more neural network models, anyways issue assigned to you @taruarora

**Comment by abhisheks008** [comment]
Updates please @taruarora

**Comment by abhisheks008** [comment]
@taruarora updates please.

**Comment by ayushete02** [comment]
:white_check_mark: 
- Full name : Ayush Shete
- GitHub Profile Link : https://github.com/ayushete02
- Approach for this Project : I will done this by using python 
- What is your participant role? 
Developer

**Comment by taruarora** [comment]
hii Ayush, I used to do AI-ML at beginner level, I would like to have your
assistance in this.
happy working together
Thanks and Regards
Taru Arora

**Comment by ayushete02** [comment]
Sure we will work together

**Comment by emojivibe** [comment]
Full name : Urvashi
GitHub Profile Link : https://github.com/emojivibe
Email ID : blueberryurvashi5@gmail.com
Approach for this Project : using a knn model
What is your participant role? contributor in SWOC 3.0

**Comment by abhisheks008** [comment]
Issue assigned to you @emojivibe

**Comment by emojivibe** [comment]
Full name : Urvashi
GitHub Profile Link : https://github.com/emojivibe
Email ID : [blueberryurvashi5@gmail.com](mailto:blueberryurvashi5@gmail.com)
Approach for this Project : tried using tensorflow transfer learning with inception v3 using keras
What is your participant role? contributor in SWOC 3.0
